Lib/test/test_c_locale_coercion.py

```diff
@@ -21,6 +20,8 @@
 # AIX uses iso8859-1 in the C locale, other *nix platforms use ASCII
```

Comment:
Rather than either letting it get out of date, or having it repeat the conditional logic as it does now, this comment can be simplified to say something generic like "While most *nix platforms default to ASCII in the C locale, some use a different encoding"

Reply:
Ok